wagtail/admin/templates/wagtailadmin/pages/side_panels/includes/page_workflow_status.html

```diff
@@ -92,6 +92,9 @@
                         {{ workflow_state.current_task_state.task.name }}
                         {{ workflow_state.current_task_state.started_at|naturaltime }}
                     {% endif %}
+                    {% if workflow_state.requested_by %}
+                        by {{ workflow_state.requested_by|user_display_name }}
+                    {% endif %}
```

Comment:
Few things here:

- There is a `since_text` variable on there that doesn’t seem to be used?
- The `by {{ workflow_state.requested_by|user_display_name }}` needs to be translate-able

Overall it’d be better if we made the whole thing translateable as a single string (with `blocktrans`). Is that possible?

Reply:
It's interesting to me that we would want to translate a users display name. Isn't this a user entered name so they would have entered the value for their name for example i want my username to be potato but i'm german so i make it Kartoffel

Reply:
But I guess it's how the name is used in the context likely 🤔 ? haha either way I shall try this all inside of a `blocktrans`

Reply:
I've tried to wrap in a `blocktrans` but I found 2 issues
1. `blocktrans` wouldn't allow if statements to be inside
2. wrapping `{{ workflow_state.requested_by|user_display_name }}` in a block trans made it disappear 👀

Reply:
We should have resolved this now with our pair programming session on Friday 🎉  so marking this thread as resolved now